### DISCUSSION TITLE
Extended sRGB support

### DISCUSSION THREAD

**Discussion by qmathe** [Ogeon/palette#298] [Q&A] [answered]
Hi,

I'm wondering whether palette supports extended sRGB (linear or non-linear) as ColorSync does on macOS. I want to create new wide-gamut colors with palette, convert them to linSrgb structs with values outside [0, 1] range, then convert them to Display P3 using ColorSync. Is that possible?

Cheers,
Quentin.

**Comment by Ogeon** [comment, marked as answer]
The library already support values outside that range, but converting between color spaces will clamp them unless you call one of the `*_unclamped` functions. I'm not familiar with the details of extended sRGB, but things like adding a new set of primaries or transfer function is made to be pretty simple.

I'm not sure if this answers your question exactly...

**Reply by qmathe** [reply]
ok, thanks a lot. That does answer it! I'll give a try to the unclamped functions together with `linSrgb` and see whether it does work for my use case. If that doesn't work out, I could implement a custom color space representing extended linear sRGB.

**Reply by Ogeon** [reply]
Sounds like a plan! Don't hesitate to get back if there are more questions.
